File: javascript/palindromeNumber.js
// You can see the challenge prompt at the following link below:
// https://leetcode.com/problems/palindrome-number/description/

var isPalindrome = function(x) {
    const numString = x.toString();
    const reversedString = numString.split('').reverse().join('');
    return numString === reversedString;
};

console.log(isPalindrome(202)); // true

File: python/palindromeNumber.py
# You can see the challenge prompt at the following link below:
# https://leetcode.com/problems/palindrome-number/description/

class Solution:
    def isPalindrome(self, x: int) -> bool:
        str_x = str(x)
        return str_x == str_x[::-1]
    
sol = Solution();
result = sol.isPalindrome(121);
print(result); # true